src/components/VideoPlayerPortalScreen/index.js

```diff
@@ -1,14 +1,14 @@
 import { Component } from 'react'
 import ReactDOM from 'react-dom'
-import { instanceOf } from 'prop-types'
+import { object } from 'prop-types'
 
 import { PROP_TYPE_CHILDREN } from '../constants'
 
 
 export default class VideoPlayerPortalScreen extends Component {
   static propTypes = {
     children: PROP_TYPE_CHILDREN,
-    videoRoot: instanceOf(Element),
+    videoRoot: object,
```

Comment:
`Element` doesn't always exist (like in SSR).  this isn't great, but it's better than failing